sdk/python/kfp/_client.py

```diff
@@ -494,63 +579,6 @@ def __repr__(self):
     run_info = self.run_pipeline(experiment.id, run_name, pipeline_file, arguments)
     return RunPipelineResult(self, run_info)
 
-  def schedule_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None,
```

Comment:
discussion: shall we write a shim that keeps the old interface and let it use the new API while providing a deprecation notice? Looking at https://github.com/kubeflow/pipelines/issues/3523, maybe there are people already using it.

@Ark-kun @faweis @numerology

Reply:
Well it seems as it doesn't work for those people anyway.
But of course it could be done, but then I would also add `start_time=None, end_time=None` as function arguments.

Reply:
Yep, sounds reasonable.